Tutorials/Template_For_Notebooks.ipynb

```diff
@@ -6,10 +6,9 @@
    "source": [
```

Comment:
<p>Explicitly mentioning the version of the kernel is a bit annoying, as one needs to update this information regularly. Is there a good reason not to simple load the default kernel <code>conda/analysis3</code> by default ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/167/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
You are right. 

Well what we'd like is that we have some way to see with which kernel the notebook was created (so that we may be able to debug things that don't work in the future). Right? So what's the best way we do that?

Reply:
<p>Perhaps we simply drop the line "Requirements: ...."?</p><p> </p><p>Can we enforce the template to open with whatever is the stable environment at the time?</p>

Reply:
<p>Yes! Just load <code>conda/analysis3</code></p><p> </p><p>Note however that using this will create a different problem: as the stable environment gets updated quarterly, some notebooks might stop working correctly after an update. I guess that should not happen very often and is probably much better than having the explicit kernel version, as those get deleted every 9 months.</p>